lib/interactive.dart

```diff
@@ -201,7 +201,13 @@ class Interactive {
     for (var i = 0; i < xAxisCategories.length; ++i) {
       var colSVG = svg.SvgElement.tag("g");
 
-      var colData = peopleByLabel[xAxisCategories[i].value];
+      // cloning list of people to sort
+      var colData = peopleByLabel[xAxisCategories[i].value]
```

Comment:
Could you type this please? I think it's `List<model.Person>`?

Reply:
Fixed